exchanges/binance/binance_test.go

```diff
@@ -2761,11 +2744,11 @@ func TestFetchExchangeLimits(t *testing.T) {
 	t.Parallel()
 	limits, err := b.FetchExchangeLimits(t.Context(), asset.Spot)
```

Comment:
```suggestion
	l, err := b.FetchExchangeLimits(t.Context(), asset.Spot)
```
Looks like you meant this here, since it's used below and the build is currently broken